### PR TITLE
/core/stories refresh

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied rebrand as per [doc](https://docs.google.com/document/d/1mXnYnpaCJznDLtYn9Q74aVeEI3iFxVOMzF0pdL7wdu0/edit) and [design](https://www.figma.com/file/QW6HKqDBbdNPdMfUaaEXV8/24.04-Core-Success-Stories?node-id=53-2047&mode=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13854.demos.haus/core/stories
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10478

### CODE REVIEW

**Review by akbarkz — APPROVED** [review]

**Comment by juanruitina** [comment]
I applied myself the changes agreed with Lyubo, there was a little misunderstanding on our side, feel free to tweak as needed. I also unhid the logos on mobile (no good reason why we should hide them, particularly given how compact they are), and changed a bit the responsiveness to align with https://ubuntu.com/download/alternative-downloads. This seems to be a pattern that might be worth standardising, given that we are using it in other places.

Some things that still need tweaking:

- [ ] HRs and navigation show on top of the side navigation on mobile, maybe a `z-index` issue?
- [ ] In my screen resolution, Robotics doesn't get highlighted on the side navigation, neither by scrolling nor by clicking. Similarly, the "Industrial IoT" item never gets highlighted again after going past it the first time.

File: templates/core/stories.html [hunk 1]
```diff
@@ -1,386 +1,503 @@
...
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h3 class="p-heading--4">Rigado cuts customers’ time to market with Ubuntu Core</h3>
-      <p>In the fast-paced world of IoT, being able to reduce time-to-market is a priority. Rigado’s core mission is to provide scalable and secure infrastructure for their customers’ commercial IoT deployments.</p>
-      <p>
-        <a href="/engage/case-study-rigado">Rigado cuts customers’ time-to-market with Ubuntu Core and AWS&nbsp;&rsaquo;</a>
+  <section class="p-section--hero">
+    <div class="u-fixed-width">
+      <h1 class="p-text--small-caps">Celebrating smart innovation</h1>
```
Comment (lyubomir-popov) [inline templates/core/stories.html:13]:
```suggestion
      <h1 class="p-text--small-caps u-no-margin--bottom">Celebrating smart innovation</h1>
```

Comment (lyubomir-popov) [reply templates/core/stories.html:13]: can we remove the margin-bottom pls

**Review by lyubomir-popov — COMMENTED** [review]
one suggestion in the code

**Comment by lyubomir-popov** [comment]
Can we please retain the layout of success stories on medium screens like so:
image

File: requirements.txt [hunk 2]
```diff
@@ -12,7 +12,7 @@ Flask-OpenID==1.3.0
 feedgen==0.9.0
 feedparser==6.0.10
 pymacaroons==0.13.0
-requests==2.28.2
+requests==2.31.0
```
Comment (carkod) [inline requirements.txt:15]: I think this can actually be the cause of the other checks not passing. Can you try reverting it, see if it works?

Comment (mtruj013) [reply requirements.txt:15]: This was not was causing the action to fail, it was a potential fix mentioned in the original issue that I was trying. Not needed anymore though!

**Pull request merged**